package-lock.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
This file, on the root of the project I think is useless. Only inside the `ui` directory is necessary

Reply:
done